InnerEye/ML/runner.py

```diff
@@ -306,6 +306,7 @@ def after_submission_hook(azure_run: Run) -> None:
                 # be necessary if the innereye package is installed. It is necessary when working with an outer project
                 # and InnerEye as a git submodule and submitting jobs from the local machine.
                 # In case of version conflicts, the package version in the outer project is given priority.
+                logging.info(f"Attempting to merge the following conda files: {source_config.conda_dependencies_files}")
```

Comment:
Not suggesting changing it here because it's not important, but FYI: https://dev.to/izabelakowal/what-is-the-best-string-formatting-technique-for-logging-in-python-d1d

Reply:
Interesting article! Will definitely bare that in mind in the future, especially if logging more computationally-intensive data